pkg/bpmn_engine/engine_usertasks_test.go

```diff
@@ -20,3 +21,31 @@ func Test_user_tasks_can_be_handled(t *testing.T) {
 	then.AssertThat(t, instance.state, is.EqualTo(process_instance.COMPLETED))
 	then.AssertThat(t, cp.CallPath, is.EqualTo("user-task"))
 }
+
+func Test_user_tasks_can_be_continue(t *testing.T) {
+	// setup
+	bpmnEngine := New("name")
+	process, err := bpmnEngine.LoadFromFile("../../test-cases/simple-user-task.bpmn")
+	then.AssertThat(t, err, is.Nil())
+	cp := CallPath{}
+
+	instance, _ := bpmnEngine.CreateInstance(process.ProcessKey, nil)
+
+	userConfirm := false
+	bpmnEngine.AddTaskHandler("user-task", func(job ActivatedJob) {
+		if userConfirm {
+			cp.CallPathHandler(job)
+		}
+	})
+	bpmnEngine.RunOrContinueInstance(instance.instanceKey)
+
+	time.Sleep(3 * time.Second)
```

Comment:
this time.Sleep() is IMHO not necessary and just delays the overall test execution.
can we remove it?

Reply:
yes，I just use it to simulation a action